typescript-version/src/@core/utils/create-emotion-cache.ts

```diff
@@ -1,5 +1,9 @@
 import createCache from '@emotion/cache'
 
 export const createEmotionCache = () => {
-  return createCache({ key: 'css' })
+  const cache = createCache({ key: 'css' })
+
+  // cache.compat = true;
+
+  return cache
```

Comment:
I believe you did this change for tests purpose. Can you revert it as it is not necessary?
```suggestion
  return createCache({ key: 'css' });
```